reference-implementation/lib/helpers/webidl.js

```diff
@@ -0,0 +1,203 @@
+'use strict';
+const utils = require('../../generated/utils.js');
+const { rethrowAssertionErrorRejection } = require('./miscellaneous.js');
+
+// https://heycam.github.io/webidl/#new
+// https://github.com/jsdom/webidl2js/issues/193
+exports.webidlNew = (globalObject, typeName, implModule) => {
+  if (globalObject[utils.ctorRegistrySymbol] === undefined) {
+    throw new Error('Internal error: invalid global object');
+  }
+
+  const ctor = globalObject[utils.ctorRegistrySymbol][typeName];
+  if (ctor === undefined) {
```

Comment:
Maybe this should be an assert()?

Reply:
This moved into webidl2js so now it's not our problem :)